### PR TITLE
node: add knex example to sample

### PR DESCRIPTION
**Pull request by aabmass**
Depends on #26. 

This adds a server using knex that does the same thing as the sequelize one. Because of #25, **this won't work with node versions older than 14.5.0**.

Also, updated `package.json` to install dependencies from tarball, which is works like installing from npm registry directly.

### CODE REVIEW

**Review by aabmass — COMMENTED** [review]
I need to update the README still

**Comment by aabmass** [comment]
Ready for review :)

File: nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/knexConnection.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const { wrapMainKnexAsMiddleware } = require("@google-cloud/sqlcommenter-knex");
+const Knex = require("knex");
+
+const sqlcommenterMiddleware = wrapMainKnexAsMiddleware(
+  Knex,
+  {
+    client_timezone: true,
+    db_driver: true,
+    route: true,
```
Comment (odeke-em) [inline nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/knexConnection.js:9]: This are supposed to be optional, let's add a disclaimer here that adding these fields isn't mandatory and will cause a high cardinality burst. Actually I'd start with firstly putting the most important fields `traceparent` and `tracestate` then these 3 below them after a newline, and with the comment.

**Review by odeke-em — CHANGES_REQUESTED** [review]
Thank you @aabmass! LGTM, but one suggestion about explicitly giving disclosure to users lest they'll copy and paste examples and have a high cardinality burst then be surprised.

File: nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/knexConnection.js (hunk 1 above)
Comment (aabmass) [reply nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/knexConnection.js:9]: Ok will do. The actual code defaults should probably be updated, too:
https://github.com/google/sqlcommenter/blob/1bc1381d838c3f61f7a290ed757daf9cc2ef8870/nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/index.js#L18-L22

Comment (aabmass) [reply nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/knexConnection.js]: let me know how it looks now

**Comment by aabmass** [comment]
Here is the PR that updates the documentation https://github.com/orijtech/sqlcommenter-website/pull/72

**Review by balachandr — APPROVED** [review]

**Comment by aabmass** [comment]
@balachandr fixed the conflicts

**Comment by balachandr** [comment]
👍

**Pull request merged**